tests/augmentation/test_container.py

```diff
@@ -516,6 +516,40 @@ def test_class(self, device, dtype):
         _, out_labels = aug(img, labels)
         assert labels is out_labels
 
+    @pytest.mark.parametrize("B", [None, 1, 3, 5])
+    @pytest.mark.parametrize("C_i", [1, 3])
+    @pytest.mark.parametrize("C_m", [None, 1])
+    @pytest.mark.parametrize("keepdim", [True, False])
+    def test_masks_without_channel_dim(self, device, dtype, B, C_i, C_m, keepdim):
+        img_shape = (B, C_i, 1000, 500) if B else (C_i, 1000, 500)
+        mask_shape = tuple(x for x in (B, C_m, *img_shape[-2:]) if x)
+
+        inp = torch.randn(*img_shape, device=device, dtype=dtype)
+        mask = torch.randint(0, 5, mask_shape, device=device, dtype=dtype)
+
+        aug = K.AugmentationSequential(
+            # K.ImageSequential(K.RandomAffine(360, p=1.0)),
+            # K.AugmentationSequential(
+            #     K.RandomAffine(360, p=1.0),
+            #     data_keys=["input", "mask"],
+            # ),
+            K.RandomAffine(360, p=1.0),
+            data_keys=["input", "mask"],
+            keepdim=keepdim,
+        )
+
+        aug.keepdim = keepdim  # FIXME(@shijianjian): This shouldn't be necessary
```

Comment:
is this a known bug, or should it be working?

Also, if we use inside the `AugmentationSequential` items like `ImageSequential` and `AugmentationSequential` looks like they do not receive the `keepdim` even with the manual set

Reply:
Either way is fine I think. Probably remove it since I think a user may expect 3D in 3D out, or 4D in 4D out. It might be redundant.

Reply:
without this manual set after initializing the `AugmentationSequential` the behavior didn't work...